README.md

```diff
@@ -906,6 +906,7 @@ Here's the full set of configurations:
 trakt:                                        # Opt
   client_id: #####                            # Req - Trakt application client ID
   client_secret: #####                        # Req - Trakt application client secret
+  auto_refresh_token: true                    # Req
```

Comment:
Might be ideal to make it optional with the default being `false`? Would probably be less disruptive to existing configs

Reply:
I was 50/50 on this, but `false` does sound like the better option.

Reply:
check_for_attribute will automatically add it to the config if its not there but it should probably default to false for the exact reason @burkasaurusrex said